.jscsrc

```diff
@@ -1,5 +1,5 @@
 {
 	"preset": "jquery",
-
+	"maximumLineLength": 110,
```

Comment:
Do not change jscsrc, it was already defined correctly - http://contribute.jquery.org/style-guide/js/#spacing

> Lines should be no longer than 80 characters, and must not exceed 100 (counting tabs as 4 spaces). There are 2 exceptions, both allowing the line to exceed 100 characters: